src/lib/ecore_con/Ecore_Con.h

```diff
@@ -96,38 +96,38 @@
 /**
  * @defgroup Ecore_Con_Buffer Ecore Connection Buffering
  * @ingroup Ecore_Con_Group
- * 
```

Comment:
this blank space changes ...

Reply:
:(

Reply:
[:(](https://github.com/expertisesolutions/efl/pull/182/commits/68b1fb2fca9d8712c983c619b3b59c22d90dc8ac)